### DISCUSSION TITLE
Is it possible to fix it?

### DISCUSSION THREAD

**Discussion by DedaL54** [flybywiresim/aircraft#6892] [Ideas]
Quite by accident I moved the throttle lever to the reverse position and I heard thunderous engines and shifting the formwork .... Yes, that’s normal, but not in the cockpit when it’s cold and dark. I just "got in" and as I said, I accidentally moved the lever. I don't think it's a lot of work to correct that and disable an activity that is not logical ?!
The next thing that comes to mind is the question: How did I even get to the cockpit of the A 320 if there are no stairs or jetway ?? I “sit in a chair” and watch in the distance the jetway with the first door closed. It also doesn’t make sense to me and I don’t think it requires much to fix in the name of reality ?! 
What do you think about that? Of course, everything else is perfect and I really enjoy "flying"!

**Comment by hotshotp** [comment]
Woops, interesting thing about the reversers. I'll take a look 🧐😳

**Reply by DedaL54** [reply]
Any news on this?

**Reply by DedaL54** [reply]
Maybe it would be nice to have someone invited to give any answer ... say out of decency?

**Reply by hotshotp** [reply]
Dude, I'm working on it. It'll be part of a bigger update. Be patient please.

**Reply by DedaL54** [reply]
I'm sorry, I didn't mean anything bad ... I appreciate what you're doing, I won't bother you anymore!
